sycl/test/regression/bit_cast_unit32_to_half2.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: %clangxx -fsycl -fpreview-breaking-changes -c %s -o %t.out
```

Comment:
Technically we should write to /dev/null instead of a temporary file.
```suggestion
// RUN: %clangxx -fsycl -fpreview-breaking-changes -c %s -o /dev/null
```